hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/SnapshotQuotaObserverChore.java

```diff
@@ -168,7 +168,7 @@ Multimap<TableName, String> getSnapshotsToComputeSize() throws IOException {
     filter.addTypeFilter(QuotaType.SPACE);
     try (Admin admin = conn.getAdmin()) {
       // Pull all of the tables that have quotas (direct, or from namespace)
-      for (QuotaSettings qs : QuotaRetriever.open(conf, filter)) {
+      for (QuotaSettings qs : QuotaRetriever.open(conn, filter)) {
```

Comment:
Would you mind also promoting uses of the `QuotaRetriever` object up to try-with-resource blocks? In this particular case, it looks like we never close the object.

Reply:
Yep, you got it!